app/javascript/packs/components/Banner.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import FilterBar from './FilterBar';
+import PropTypes from 'prop-types';
+import AddNewRestaurant from './AddNewRestaurant';
+
+class Banner extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchValue: '',
+      openDialog: false,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange(event) {
+    event.preventDefault();
+    this.setState({ searchValue: event.target.value });
+    this.props.filterRests("search", event.target.value);
+  }
+  render() {
+    const cuisines = [];
+    Object.keys(this.props.cuisines).forEach(function (key) {
+      cuisines.push(key);
+    });
+    cuisines.sort();
+    const updateRests = this.props.updateRests;
```

Comment:
use syntax: const { updateRests } = this.props;

Reply:
👍